pulp_puppet_tools/pulp_puppet/tools/puppet_module_builder.py

```diff
@@ -25,6 +25,8 @@
 
 TAG = _('the name of a git tag to be checked out.')
 
+FORCE = _('overwrite modules in the output dir if true')
```

Comment:
Using the word `true` here seems inappropriate since the user either includes the `-f` option or does not.

Suggest: *overwrite modules in the output directory.* instead.